docs/api-setup.md

```diff
@@ -0,0 +1,68 @@
+---
+id: api-setup
+title: API Setup
+---
+
+This page will show you how to setup API requests with Postman and Paw. The main goal here is to setup OAuth2 authorization. 
```

Comment:
Since this is the very first paragraph before diving into Postman and Paw details, consider briefly introducing what these apps are (e..g `...how to set up API requests in API development tools such as Postman and Paw. The main...`)

Reply:
Got it! I'll also add a note that this process works for remote instances as well, not just `localhost`.